src/tests/base_kubeapi_test.py

```diff
@@ -169,7 +169,6 @@ def set_single_node_ip(cls, cluster_deployment: ClusterDeployment, nodes: Nodes)
         nodes.controller.tf.change_variables(
             {
                 "single_node_ip": single_node_ip,
```

Comment:
not sure why we need to run this function , it is relevant only for bootstrap_in_place test

Reply:
Updated and removed the call for `set_single_node_ip` in kube-api function